app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -251,14 +251,12 @@ def line_53c_checkbox
       end
 
       def line_59_primary
-        get_personal_excess(@intake.primary.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX) +
-          get_personal_excess(@intake.primary.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
+        get_personal_excess(@intake.primary.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX)
       end
 
       def line_59_spouse
         if @intake.filing_status_mfj?
-          get_personal_excess(@intake.spouse.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX) +
-            get_personal_excess(@intake.spouse.ssn, :box14_ui_hc_wd, EXCESS_UI_WF_SWF_MAX)
+          get_personal_excess(@intake.spouse.ssn, :box14_ui_wf_swf, EXCESS_UI_WF_SWF_MAX)
```

Comment:
review comment: instead of passing excess type, could just pass the value itself

Reply:
@mmazanec22 actually, i looked at this again and because `get_excess_value()` is called within two different iterators (ie needs to look at several w2s), it actually makes that function not so easy to get rid of...

Reply:
@mmazanec22  aha! I tried again and was able to make it work by figuring out lambda syntax better in this commit https://github.com/codeforamerica/vita-min/pull/5288/commits/f33f5e5abdfeae9f8365406426c9b7937551edf1